packages/documentation/.storybook/manager-head.html

```diff
@@ -1,7 +1,65 @@
-<!--
-  Do whatever you can do in an normal html header.
-  Storybook will inject these tags into the Storybook application UI.
--->
+<script src="/assets/scripts/analytics-helper.js"></script>
+
+<script data-id="GtmConstants">
+  window.gtm.setConstant('ENVIRONMENTS', {
+    dev: '%STORYBOOK_GTM_PAGE_CONTEXT_ENVIRONMENT_DEV%',
+    int: '%STORYBOOK_GTM_PAGE_CONTEXT_ENVIRONMENT_INT%',
+    prod: '%STORYBOOK_GTM_PAGE_CONTEXT_ENVIRONMENT_PROD%',
+  });
+
+  window.gtm.setConstant('ENVIRONMENT_FALLBACK', '%STORYBOOK_GTM_PAGE_CONTEXT_ENVIRONMENT_FALLBACK%');
+</script>
+
+<script data-id="GoogleTagManager">
+  // Base
+  (function (w, d, s, l, i) {
+    w[l] = w[l] || [];
+    w[l].push({ 'gtm.start': new Date().getTime(), 'event': 'gtm.js' });
+
+    let f = d.getElementsByTagName(s)[0],
+      j = d.createElement(s),
+      dl = l != 'dataLayer' ? '&l=' + l : '';
+
+    j.async = true;
+    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
+    f.parentNode.insertBefore(j, f);
+  })(window, document, 'script', 'dataLayer', '%STORYBOOK_GTM_KEY%');
+
+  window.dataLayer = window.dataLayer || [];
+
+  // Page Context
+  window.addEventListener('storybook:ready', function () {
+    window.dataLayer.push({
+      event: 'page_context', // constant, required
+      spa: 'true', // constant, required
+      content_language: '%STORYBOOK_GTM_PAGE_CONTEXT_CONTENT_LANGUAGE%', // required
+      content_geo_region: '%STORYBOOK_GTM_PAGE_CONTEXT_CONTENT_GEO_REGION%', // required
+      source_code_version: '%STORYBOOK_GTM_PAGE_CONTEXT_SOURCE_CODE_VERSION%', // recommended
+      page_name_language_neutral: window.gtm.getPageTitle(), // required
+      page_url_language_neutral: window.location.href, // required
+      environment: window.gtm.getEnvironment(), // required
+      login_status: 'false', // required
+      platform_name: 'ms_design_system', // required
+      department_name: 'IT16.12', // required
```

Comment:
Maybe we can create an issue to remember to change this variable?

Reply:
It's already getting the correct value (`dev`, `int`, `prod`) by checking for the used url.